ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -1479,11 +1479,13 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
       E2E_RUN_TAGS: '@fio'
       FILTERS_ADDITIONAL: ""
+      INDEX_IMAGE: quay.io/redhat-user-workloads/ocp-isc-tenant/file-integrity-operator-fbc-4-17@sha256:15b55610bbf6bb442a07850272d35f4a9d96aa2d5cc02e8695a03bb821bd5d17
```

Comment:
Should be able to update this to:

[quay.io/redhat-user-workloads/ocp-isc-tenant/file-integrity-operator-fbc-4-17@sha256:37a04722d6eade1651605a417bc64eb77641d13fa79005bd0afaa24017d36492](http://quay.io/redhat-user-workloads/ocp-isc-tenant/file-integrity-operator-fbc-4-17@sha256:37a04722d6eade1651605a417bc64eb77641d13fa79005bd0afaa24017d36492)